requirements.txt

```diff
@@ -1,6 +1,6 @@
 cairosvg
 gitpython
-hilbertcurve
+hilbertcurve==1.0.5
```

Comment:
@HackerFoo Please file an issue with upstream `hilbertcurve`, and create an issue to remove the pin once upstream `hilbertcurve` is fixed.

Reply:
I don't think the issue is upstream. The API changed.

Reply:
The PR comment in https://github.com/galtay/hilbertcurve/pull/25#issue-519041227 indicated that that specific PR was supposed to be backwards compatible.  It is unclear if the API interface was supposed to change here, or if it was an accident.

Reply:
My comment was basically to post an issue upstream showing that from 1.0.5 to 2.0.x that this API no longer existed/worked, and determine if that break was intentional?

Reply:
Although the notes from 2.0.3 do say `New API`, which is foreboding.

Regardless, we should likely create an issue to remove the pin in the future to avoid being in a situation where we have a very stale dependency.  This isn't equivalent, but a numpy pin on prjxray eventually resulted in the pip install for numpy to require building it instead of using pip binary caches.  Ideally we'd like to avoid something like that.

Reply:
I propose removing the dependency and using VPR's RR node reordering option: https://github.com/SymbiFlow/symbiflow-arch-defs/issues/1773

Reply:
Sure.  Can you please open a PR to that effect?

Reply:
Yeah, I'll assign the issue to me.